src/dappsExplorer/DAppsExplorerScreen.tsx

```diff
@@ -97,7 +97,7 @@ export function DAppsExplorerScreen() {
     <SafeAreaView style={styles.safeAreaContainer} edges={['top']}>
       <DrawerTopBar
         middleElement={<Text style={headerStyles.headerTitle}>{t('dappsScreen.title')}</Text>}
-        rightElement={<TopBarIconButton icon={<Help />} onPress={onPressHelp} />}
+        rightElement={<TopBarIconButton testID="HelpIcon" icon={<Help />} onPress={onPressHelp} />}
```

Comment:
nit: DAppsExplorerScreen/HelpIcon

Reply:
Fixed in afaa22c75524205d035e80d06bc31f3fef80a8ff!